rewrite-maven/src/main/java/org/openrewrite/maven/tree/MavenRepository.java

```diff
@@ -37,7 +37,7 @@
 public class MavenRepository implements Serializable {
 
     public static final MavenRepository MAVEN_LOCAL_USER_NEUTRAL = new MavenRepository("local", new File("~/.m2/repository").toString(), "true", "true", true, null, null, false);
-    public static final MavenRepository MAVEN_LOCAL_DEFAULT = new MavenRepository("local", new File(System.getProperty("user.home") + "/.m2/repository").toURI().toString(), "true", "true", true, null, null, false);
+    public static final MavenRepository MAVEN_LOCAL_DEFAULT = new MavenRepository("local", "file://" + System.getProperty("user.home") + "/.m2/repository", "true", "true", true, null, null, false);
```

Comment:
My immediate thought here is what about Windows? From what I can find the value there is similar to
```
c:\Users\Foo
```
Would this then still work with that prefix? Would the same test pass or fail on Windows?

Reply:
I suspect it will be safest to create a `Path` and then an URI from that using `toURI()`.

Reply:
The original implementation or using `Path` are the only correct mechanisms to get a valid URI for all operating systems.

On Mac and *nix, the uri looks like:
```
# macOS
user.home=/Users/Foo
file:///Users/Foo/.m2/repository

# *nix
user.home=/home/Foo
file:///home/Foo/.m2/repository
```

For windows, it looks like:
```
user.home=C:\Users\Foo
file:///C:/Users/Foo/.m2/repository
```

By using `File` or `Path` as an intermediate step ensures that the result URI is valid for all operating systems.

Reply:
This indeed seems to work for me on linux; anyone able to try the same on Windows?
```
jshell> Paths.get(System.getProperty("user.home"), ".m2", "repository").toUri().toString()
$1 ==> "file:///home/tim/.m2/repository/"
```

```suggestion
    public static final MavenRepository MAVEN_LOCAL_DEFAULT = new MavenRepository("local", Paths.get(System.getProperty("user.home"), ".m2", "repository").toUri().toString(), "true", "true", true, null, null, false);
```

Reply:
From Windows:
```
jshell> Paths.get(System.getProperty("user.home"), ".m2", "repository").toUri().toString();
$2 ==> "file:///C:/Users/tso4045/.m2/repository/"
```